### ISSUE TITLE
Configure rack map cabinet labels to show owner or other attributes

### ISSUE THREAD

**Issue by eschoeller** [samilliken/openDCIM#1024]
I'd like to have the automated cabinet labels on the map be configured to show either the cabinet location (as it does today) or the cabinet owner (or possibly other cabinet attributes as well). Our cabinet locations never really change, so I've statically drawn these into our map image. What does change is who the owner is of each cabinet, and it would be ideal to have openDCIM display that instead in the label. 

Thanks for considering!

**Comment by wilpig** [comment]
You can already put that information in the tooltip

On Mar 18, 2018 9:09 PM, Eric <notifications@github.com> wrote:

I'd like to have the automated cabinet labels on the map be configured to show either the cabinet location (as it does today) or the cabinet owner (or possibly other cabinet attributes as well). Our cabinet locations never really change, so I've statically drawn these into our map image. What does change is who the owner is of each cabinet, and it would be ideal to have openDCIM display that instead in the label.

Thanks for considering!

**Comment by eschoeller** [comment]
Yep, I see it in the tool-tip, but I'd really like to have it drawn straight onto the map if possible. Right now the cabinet location is shown twice:
![image](https://user-images.githubusercontent.com/12864741/37575279-00d6a97e-2aec-11e8-80ae-9efc7d58c098.png)
I could put the owner directly on our map image but I'd have to change the image each time the customer changes (and unfortunately today this happens somewhat frequently).

**Comment by eschoeller** [comment]
Any further consideration on making this an option?
